frontend/frontend.js

```diff
@@ -57,7 +57,7 @@ function getStuff() {
                 let long = geoJSON["geometry"]["coordinates"][0]
                 let lat = geoJSON["geometry"]["coordinates"][1]
 
-                L.marker([lat,long]).addTo(map);
+                L.marker([lat,long]).bindPopup(geoJSON["properties"]["amenity"]).addTo(map);
```

Comment:
There is a case where the properties object is missing has "leisure" instead of "amenity"